t/op/isa.t

```diff
@@ -11,7 +11,7 @@ use strict;
 use feature 'isa';
 no warnings 'experimental::isa';
 
-plan 11;
```

Comment:
Can you please give your commits more descriptive titles? «Fix github issue 18679» doesn't tell a casual reader what it happening here, «Ensure isa object method still works after using isa infix operator» does